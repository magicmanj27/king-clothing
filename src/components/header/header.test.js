import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";

import Header from "./header.component";

const mockStore = configureMockStore();
const store = mockStore({});

describe("Header Component", () => {
  it("should render without throwing an error", () => {
    expect(
      shallow(
        <Provider store={store}>
          <Header />
        </Provider>
      )
    ).toMatchSnapshot();
  });
});

// describe("if current is null", () => {
//   it("should render sign in link", () => {
//     const mockProps = {
//       currentUser: null,
//     };

//     const newWrapper = shallow(
//       <Provider store={store}>
//         <Header {...mockProps} />
//       </Provider>
//     );

//     expect(newWrapper.find("[id='test']").text()).toBe("SIGN IN");
//   });
// });
