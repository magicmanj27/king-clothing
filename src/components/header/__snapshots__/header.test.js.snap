// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render without throwing an error 1`] = `ShallowWrapper {}`;
